packages/mongodb-memory-server-core/src/util/getos/index.ts

```diff
@@ -125,9 +126,13 @@ async function getLinuxInfomation(): Promise<LinuxOS> {
  */
 async function tryLSBRelease(): Promise<LinuxOS | undefined> {
   try {
-    const lsb = await promisify(exec)('lsb_release -a'); // exec this for safety, because "/etc/lsb-release" could be changed to another file
+    // use upstream lsb file if it exists (like in linux mint)
```

Comment:
imo this will prevent us from custom handling for some derivates (tried it for mint, according to google it should work for elementary too). therefore I extended the custom parsers for elementary and mint can be dropped.

Reply:
Will try this in a vm later/tomorrow